manifests/init.pp

```diff
@@ -9,6 +9,7 @@
   $enable_fqdn_entry     = true,
   $use_fqdn              = true,
   $fqdn_host_aliases     = $::hostname,
+  $fqdn_ip_address       = $::ipaddress,
```

Comment:
The validation of `$fqdn_ip_address` needs a spec test to show that a non-ip will fail.